grammars/JSONC.arm.tmLanguage.json

```diff
@@ -2,11 +2,11 @@
     "$modified_from": "https://github.com/Microsoft/vscode/blob/6850012cf41f74fc48bf025569c0ec383c341df9/extensions/json/syntaxes/JSONC.tmLanguage.json",
```

Comment:
These changes are about colorization. Wouldn't spend a bunch of time reviewing, there are plenty of tests.